recipes/boost/all/conanfile.py

```diff
@@ -1109,7 +1116,8 @@ def add_defines(library):
             if self.options.multithreading:
                 cxx_flags.append("-DBOOST_SP_USE_SPINLOCK")
 
-            cxx_flags.append("-fembed-bitcode")
+            if "embed_bitcode" in self.options and self.options.embed_bitcode:
```

Comment:
Could it be simplified by:

```suggestion
            if self.options.get_safe("embed_bitcode"):
```
?